README.md

```diff
@@ -106,13 +106,13 @@ Wikibase Internal Serialization has been written by [Jeroen De Dauw]
 
 ## Release notes
 
-### 1.4 (dev)
+### 1.4 (2015-06-12)
 
-* Added PHPMD and PHPCS support
-* Added support for showing the component version when loaded via MediaWiki
-* Added compatibility with DataModel 3.x and DataModelSerialization 1.4
+* Added compatibility with DataModel 3.x
```

Comment:
I also think that it we should mention the other version changes. It also doesn't affect the user that we "Added PHPMD and PHPCS support" but we put it into the release notes anyways.